.github/workflows/gempush.yml

```diff
@@ -0,0 +1,41 @@
+name: Ruby Gem
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
This restricts the action to only run when a tag matching this glob is pushed.

Reply:
Yeah, that's what I was hoping for. I think that should match the current tag convention we've been using?